### DISCUSSION TITLE
SRCF Project webNG

### DISCUSSION THREAD

**Discussion by CHTJonas** [SRCF/projects#18] [Ideas]
To quote @mas90, "literally every aspect of sinkhole sucks" 🤣. To that end, I have drafted a proposal for how the SRCF's next generation of web hosting could be architectured. Feedback welcomed!

https://notes.srcf.net/s/ry2DHoxP_#

**Comment by mas90** [comment]
> To quote @mas90, "literally every aspect of sinkhole sucks" 🤣

Opinions expressed are solely my own and do not express the views or opinions of the SRCF or my employer. ;-)

**Comment by doismellburning** [comment]
This sounds great, nice work
